src/components/SideNav/SideNav.js

```diff
@@ -29,14 +27,6 @@ const checkIfOverview = (relativePath) => {
   return pathSegments[pathSegments.length - 1] === "index.mdx";
 };
 
-export const SideNav = ({ children, ...props }) => (
```

Comment:
I realized this is just re-exporting `StickyEl`, so i removed this, updated imports, and renamed StickyEl to `SideNavContainer`